### PR TITLE
Update description of a `credentialSubject` for a credential type `VerifiedEmployee`

### PR DESCRIPTION
**Pull request by Sakurann**
The usage of this credential type is OPTIONAL and is not required to be compliant with this profile.

### CODE REVIEW

File: spec/spec.md [hunk 1]
```diff
@@ -565,11 +565,16 @@ While ION supports any public key JWK representation in a DID Document, implemen
 Note: This profile leverages JWT for signature generation and verification only. There is a rich offering of Linked Data Cryptographic Suites which are not covered by this iteration of the profile.
 For reference and more information on LD signature suites see the [Linked Data Cryptographic Suite Registry](https://w3c-ccg.github.io/ld-cryptosuite-registry/).
 
-## Credential Schema
+## Credential type `WokrplaceCredential`
```
Comment (ibmdc) [inline spec/spec.md:568]: typo - WokrplaceCredential -> `WorkplaceCredential`  :)

Comment (Sakurann) [inline spec/spec.md:568]:
```suggestion
## Credential type `WorkplaceCredential`
```

File: spec/spec.md [hunk 2]
```diff
@@ -565,11 +565,16 @@ While ION supports any public key JWK representation in a DID Document, implemen
...
+Below is a description of a `credentialSubject` for a credential type `WorkplaceCredential`. It is RECOMMENDED to be used with a Workplace Credential Use-Case defined below. However the usage of this credential type is OPTIONAL and is not required to be compliant with this profile.
 
-REQUIRED claim is only `displayName`. All other claims are OPTIONAL and might be omitted.
+- `displayName` REQUIRED. String. The name displayed for the user. This is usually the combination of the user's first name and last name. Maximum length is 256 characters.
+- `givenName` OPTIONAL. String. The given name (first name) of the user. Maximum length is 64 characters.
+- `surname`  OPTIONAL. String. The user's surname (family name or last name). Maximum length is 64 characters.
+- `jobTitle` OPTIONAL. String. The user's job title. Maximum length is 128 characters.
+- `preferredLanguage` OPTIONAL. String. The preferred language for the user. Should follow ISO 639-1 Code, for example en-US.
+- `photo`  OPTIONAL. The user's profile photo. The data is binary and not encoded in base64.
```
Comment (quartzjer) [inline spec/spec.md:577]:
```suggestion
- `photo`  OPTIONAL. The user's profile photo. The value is a data URL as defined in RFC 2397.
```

**Review by quartzjer — APPROVED** [review]

File: spec/assets/workplace_credential_vc.json [hunk 3]
```diff
@@ -1,20 +1,19 @@
...
+      "VerifiableCredential",
+      "WorkplaceCredential"
+    ],
+    "credentialSubject": {
+      "displayName": "$.displayName",
+      "givenName": "$.givenName",
+      "jobTitle": "$.jobTitle",
+      "surname": "$.surname",
+      "preferredLanguage": "$.preferredLanguage",
+      "mail: "$.mail",
```
Comment (jischr) [inline spec/assets/workplace_credential_vc.json:15]: missing closing " on mail.

Comment (jischr) [inline spec/assets/workplace_credential_vc.json:15]:
```suggestion
      "mail": "$.mail",
```

File: spec/assets/workplace_credential_vc.json [hunk 4]
```diff
@@ -1,20 +1,19 @@
...
-    "photo": "data:image/jpeg;base64,ewrsfirGWRPrewFEW4..."
-  },
-  // VC truncated...
+"vc": {
+    "@context": [
+      "https://www.w3.org/2018/credentials/v1",
+    ],
+    "type": [
+      "VerifiableCredential",
+      "WorkplaceCredential"
```
Comment (Sakurann) [inline spec/assets/workplace_credential_vc.json:7]:
```suggestion
      "VerifiedEmployee"
```

**Comment by jischr** [comment]
whats the context for changing this to `VerifiedEmployee`? think i missed that discussion

File: spec/spec.md (hunk 2 above)
Comment (Sakurann) [reply spec/spec.md:577]: Could I externalize this comment into an issue? I think we need to agree how we want photos to be handled.

**Review by jischr — APPROVED** [review]

**Pull request merged**